common/optimiser/OptimiserConstraint.java

```diff
@@ -31,24 +31,15 @@ public class OptimiserConstraint {
     private final String name;
     private final Map<OptimiserVariable<?>, Double> coefficients;
     private MPConstraint mpConstraint;
-    boolean isUpToDate;
```

Comment:
This flag is no longer necessary, as `GraphPlanner` now signals the state changes directly to the `Optimiser`.